Formula/mu.rb

```diff
@@ -15,6 +15,7 @@ class Mu < Formula
   end
 
   depends_on "autoconf" => :build
+  depends_on "autoconf-archive" => :build if build.head?
```

Comment:
Create a `head do` block and put the `url "https://github.com/djcb/mu.git"` and `depends_on "autoconf-archive" => :build` in there.